ocaml/xapi/xapi_host.ml

```diff
@@ -3055,18 +3060,24 @@ let try_restart_device_models_for_recommended_guidances ~__context ~host =
       None
 
 let apply_recommended_guidances ~__context ~self =
```

Comment:
It's a bit confusing to use the name 'self' for an external host.
It's possible to rename self to host in the function like this:
```suggestion
let apply_recommended_guidances ~__context ~self:host =
```

Reply:
Hi @psafont , sorry, I don't know the differences between "renaming with ~self:host" and "change 'self' directly to 'host'", as this is a new added api, so in the fixup: [1979db5](https://github.com/xapi-project/xen-api/pull/4991/commits/1979db5a88bf5eb12bac8ac9e88c39a152acc439), I renamed "self" directly to "host".
Is this fix ok? Or would you please share more about differences, thank you!

Reply:
Pau's way left the name of the argument for callers of the function as `self`, while inside the function, the parameter is renamed to `host`. It is better to do that, because the argument in the API should remain `self` for consistency with the rest of the API.